docs/en/latest/plugins/jwt-auth.md

```diff
@@ -196,6 +196,20 @@ then add jwt-auth plugin in the Consumer page:
 
 #### Get the Token in `jwt-auth` Plugin:
 
+First you need to setup the route for the API that sign the token, which will use the [public-api](public-api.md) plugin.
+
+```shell
+$ curl http://127.0.0.1:9080/apisix/admin/routes/ -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/apisix/plugin/jwt/sign",
```

Comment:
Complex for me to understand 😅

Reply:
Haha, It is not necessary for you to understand its actual meaning, you can check the content such as the description in English and Chinese, and other content is checked by other reviewers.